doc/man7/provider-cipher.pod

```diff
@@ -349,6 +349,16 @@ by AES SIV ciphers which disallow multiple operations by default.
 Setting "speed" to 1 allows another encrypt or decrypt operation to be
 performed. This is used for performance testing.
 
+=item "gcm_iv_gen" (B<OSSL_CIPHER_PARAM_GCM_GET_IV_GEN>) <octet string>
+
+Gets the GCM invocation field generated for encryption. Can only be called
+after "tlsivfixed" is set.
+
+=item "gcm_iv_inv" (B<OSSL_CIPHER_PARAM_GCM_SET_IV_INV>) <octet string>
+
+Sets the GCM invocation field used for decryption. Can only be called
+after "tlsivfixed" is set.
+
```

Comment:
If gcm and ccm parameters of this sort are really AEAD parameters, why do we insist on using "gcm" in the params?  Also, if this is meant specifically for TLS use, why not name these parameters in the same manner as the other TLS specific params, such as "tlsivgen" and "tlsivinv"?  If not meant specifically for TLS use, why not "ivgen" and "ivinv"?

Reply:
updated names to tlsivgen and tlsivinv (I was just mapping to the control names)